Polyfills/Canvas/Source/Canvas.cpp

```diff
@@ -5,11 +5,17 @@
 #include <functional>
 #include <sstream>
 #include <assert.h>
+#include <NativeEngine.h>
 
 namespace Babylon::Polyfills::Internal
 {
     static constexpr auto JS_CONSTRUCTOR_NAME = "NativeCanvas";
 
+    struct NativeCanvas::Impl final
+    {
+        std::unique_ptr<Babylon::TextureData> TextureData;
```

Comment:
Do you need the unique_ptr?

Reply:
no we don't, good call out.

Reply:
this should now be fixed.

Reply:
see changes, this struct no longer exists.